src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1849,6 +1849,9 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode, callable $e
                                         ->variableNode('md5')->end()
                                     ->end()
                                 ->end()
+                                ->scalarNode('crypto_method')
+                                    ->info('The minimum version of SSL to accept; must be one of STREAM_CRYPTO_METHOD_TLSv*_CLIENT constants.')
```

Comment:
Minor suggestion:

`The minimum version of SSL to accept [...]`
->
`The minimum version of TLS/SSL to accept [...]`

TLS is the successor protocol to SSL, so I think we should mention it and do that in the first place. Thanks.

Reply:
updated, thanks for the suggestion

Reply:
I would even mention only TLS, as we don't support any SSL version for this setting.

Reply:
PR updated to tell only about "TLS"